simde/arm/neon/ld2.h

```diff
@@ -0,0 +1,66 @@
+/* SPDX-License-Identifier: MIT
+ *
+ * Permission is hereby granted, free of charge, to any person
+ * obtaining a copy of this software and associated documentation
+ * files (the "Software"), to deal in the Software without
+ * restriction, including without limitation the rights to use, copy,
+ * modify, merge, publish, distribute, sublicense, and/or sell copies
+ * of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ * BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ * Copyright:
+ *   2020      Evan Nemerson <evan@nemerson.com>
+ */
+
+#if !defined(SIMDE_ARM_NEON_LD2_H)
```

Comment:
The failure on ICC is due to a bug in the compiler where it can't handle these types.  I actually thought this was fixed in a recent version, but I just tried enabling the code which triggers the issue and there is still an ICE (I think it may be a slightly different issue, but the report was lost when Intel changed their forums… I'll report the new issue as soon as I have reduced test case).

Anyways, to work around it you can just change this to `#if !defined(SIMDE_ARM_NEON_LD2_H) && !defined(SIMDE_BUG_INTEL_857088)`, and you'll have to ifdef out the tests too.  See ld3 and/or trn for examples.  Sorry, I know it's annoying… if you want me to take care of it instead just let me know, I don't mind.